shadows/framework/src/main/java/org/robolectric/shadows/ShadowPausedSystemClock.java

```diff
@@ -108,7 +108,20 @@ private static void informListeners() {
    *
    * <p>This API sets both of the elapsed realtime and uptime to the specified value.
    *
-   * <p>Currently does not perform any permission checks.
+   * <p>Use of this method is discouraged. It currently has the following inconsistent behavior:
+   *
+   * <ol>
+   *   <li>>It doesn't check permissions. In real android this method is protected by the
+   *       signature/privileged SET_TIME permission, thus it is uncallable by most apps
+   *   <li>It doesn't actually change the value of System.currentTimeMillis for non-instrumented
+   *       code aka nearly all user tests and apps It only allows advancing the current time, not
+   *       moving it backwards
+   *   <li>It incorrectly changes the value of SystemClock.uptime, elapsedRealtime, and for
+   *       instrumented code System.nanoTime. In real android these are all independent clocks
+   * </ol>
+   *
+   * <p>It is recommended to use ShadowSystemClock.advanceBy instead to advance
+   * SystemClock.uptimeMillis and SystemClock.elapsedRealTime
```

Comment:
Adding some formatting and links to other methods:
```suggestion
   * <p>Use of this method is discouraged. It currently has the following inconsistent behavior:
   *
   * <ol>
   *   <li>It doesn't check permissions. In real Android, this method is protected by the
   *       signature/privileged {@code SET_TIME} permission, thus it is un-callable by most apps
   *   <li>It doesn't actually change the value of {@link System#currentTimeMillis()} for
   *       non-instrumented code, aka nearly all user tests and apps. It only allows advancing the
   *       current time, not moving it backwards
   *   <li>It incorrectly changes the value of {@link SystemClock#uptimeMillis()}, {@link
   *       SystemClock#elapsedRealtime()}, and for instrumented code {@link System#nanoTime()}. In
   *       real Android, these are all independent clocks
   * </ol>
   *
   * <p>It is recommended to use {@link ShadowSystemClock#advanceBy(Duration)} instead to advance
   * {@link SystemClock#uptimeMillis()} and {@link SystemClock#elapsedRealtime()}
```